crates/graphql_artifact_generation/src/format_parameter_type.rs

```diff
@@ -75,7 +75,7 @@ fn format_server_field_type(
 fn format_field_definition(
     schema: &ValidatedSchema,
     name: &SelectableFieldName,
-    type_: &FieldType<ServerFieldId, ClientType<ClientFieldId>>,
+    type_: &FieldType<ServerFieldId, ClientType<ClientFieldId, ClientPointerId>>,
```

Comment:
sorry I meant to rebase just the commits that relate to generating the schema, i.e. we shouldn't have any changes related to ☝️